### DISCUSSION TITLE
在不修改preload的情况下，如何注入electron的渲染进程？

### DISCUSSION THREAD

**Discussion by FlysoftBeta** [ShellWen/v8_killer#6] [Q&A]
rt

**Comment by ShellWen** [comment]
目前没有做区分 context，唯一的方法是将原有的 preload 的文件名写进 keyword-matcher，然后使用 v8 killer 修改 preload，局限性也很明显了，如果没有加载 preload 的话，就无法修改

当然你也可以修改 electron，在 electron 模块后添加代码做 monkey patch
